### DISCUSSION TITLE
M

### DISCUSSION THREAD

**Discussion by 37elivan73** [community/community#147132] [Programming Help]
### Select Topic Area

Bug

### GitHub Feature Area

Actions

### Body

Hello, I have a problem. They have me with a security privacy which is making me lose everything because as you can see it makes me look 40 years younger which disables me for committing and like everyone else I live my life as What is it please help

**Comment by mecodeatlas** [comment]
Hi @37elivan73 , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
